addons/website_sale/models/product_image.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields, api
+
+
+class ProductImage(models.Model):
+    _name = 'product.image'
```

Comment:
website_sale.image ?

Reply:
no, will be moved in futur to product module probably.
(for online quote and other module)